### ISSUE TITLE
Weird sentence

### ISSUE THREAD

**Issue by rhiaro** [w3c/activitystreams#384]
Please Indicate One: 
 * [X] Editorial
 * [ ] Question
 * [ ] Feedback
 * [ ] Blocking Issue
 * [ ] Non-Blocking Issue

> If there is no name property, the summary property SHOULD NOT include markup, and SHOULD be as short enough to be used as a text representation in the language context.

I would rephrase to:

> If there is no name property, the summary property SHOULD NOT include markup, and SHOULD be short enough to be used as a reasonable text representation of the object.

@evanp If you'd like me to just push this change in, say yes.

**Comment by evanp** [comment]
yes

**Comment by rhiaro** [comment]
The most confusing part of the sentence to me is "in the language context" and this hasn't been updated..

**Comment by evanp** [comment]
Updated it to your exact wording.
